src/redteam/providers/advNoise.ts

```
@@ -0,0 +1,322 @@
+function addPunctuationJitter(text: string, punctuationRate: number = 0.1): string {
+  const punctuation = ['.', ',', '!', '?', ';', ':'];
+  const chars = text.split('');
+  const numChanges = Math.floor(text.length * punctuationRate);
+
+  for (let i = 0; i < numChanges; i++) {
+    const pos = Math.floor(Math.random() * chars.length);
+    const changeType = Math.floor(Math.random() * 3);
+
+    switch (changeType) {
+      case 0: // Add punctuation
+        const randomPunct = punctuation[Math.floor(Math.random() * punctuation.length)];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)